### ISSUE TITLE
Published posts are unable to be unpublished if the source note is deleted

### ISSUE THREAD

**Issue by EnMod** [standardnotes/listed#9]
This, I assume, is because Listed makes a sort of copy of the post content when you publish, and the unpublish command in each note is tied to that note's unique ID.  So, once that note is deleted so is the ability to unpublish it.  

Makes sense in theory, but I didn't really have it figured out when I deleted the notes for some posts I published while testing custom CSS.  In hindsight I should have published privately...but anyway: 

**How do I delete posts published from now-deleted notes? Is this doable or am I stuck with two test posts for the life of my Listed page?**  I have little to no subscribers at present, so I suppose I could just generate a new author link and use that...but if I use the same username and such wouldn't that clash with the other version of my blog?

**Comment by mobitar** [comment]
This is currently a limitation. Although with the new metatype format, perhaps we can publish commands. 

Something like:

```
---
metatype: command
delete_post_by_title: title
---
```

This could be a good solution. For now, I've went ahead and deleted your test posts.

**Comment by EnMod** [comment]
That's an interesting solution, I'm digging these metatypes and what they can do.  Interested to see what can be done with the command type.

Also thanks for taking care of that...I should have known about your ability to admin posts hosted on Listed itself, but up until you mentioned it I was completely unaware.

Anyway I think that's all for this issue 👍 thanks again for the prompt response!

**Comment by EnMod** [comment]
Actually, maybe I should leave this open until the ability to send deletion commands is implemented? So other users are aware of the behavior.

**Comment by mobitar** [comment]
Yup that's cool. And about metatypes, the next immediate thing I want to do is allow for a metatype of "json", so a user can publish a json snippet and the server will render it the correct content-type header. Useful for publishing to an API, but more directly, the new Standard Notes package format relies on a JSON endpoint to deliver extension information and autoupdate information, so it will allow users to avoid creating their own server to install their own packages.
